Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -2261,6 +2263,16 @@ namespace Babylon
         }
     }
 
+    void NativeEngine::PopulateFrameStats(const Napi::CallbackInfo& info)
```

Comment:
Is the thinking we might add more stats later? If not, we could just return a number directly.

Reply:
Yes that was my thinking. I will switch to a number if you object, otherwise I'll leave it as is.

Reply:
I'm not objecting, just asking.